src/main/kotlin/com/openlattice/auditing/LocalAuditingService.kt

```diff
@@ -0,0 +1,104 @@
+package com.openlattice.auditing
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.google.common.collect.ArrayListMultimap
+import com.google.common.collect.ImmutableMap
+import com.openlattice.data.DataEdge
+import com.openlattice.data.DataGraphManager
+import com.openlattice.data.EntityDataKey
+import java.util.*
+
+class LocalAuditingService(
+        private val dataGraphService: DataGraphManager,
+        private val ares: AuditRecordEntitySetsManager,
+        private val mapper: ObjectMapper
+) : AuditingManager {
+
+    fun recordEvent(event: AuditableEvent): Int {
+        return recordEvents(listOf(event))
+    }
+
+    override fun recordEvents(events: List<AuditableEvent>): Int {
+
+        val auditingConfiguration = ares.auditingTypes
+
+        return if (auditingConfiguration.isAuditingInitialized()) {
```

Comment:
Any chance you'd be willing to this instead:
```
if ( !auditingConfiguration.isAuditingInitialized() ){
    return 0
}
// CODES
// return things
```

Reply:
there is
a chance

Reply:
![phew](https://res.cloudinary.com/teepublic/image/private/s--B1PC_Ipd--/t_Preview/b_rgb:ffffff,c_limit,f_jpg,h_630,q_90,w_630/v1504759721/production/designs/1878951_1.jpg)